data-pipeline/src/pipeline/mappings.py

```diff
@@ -164,19 +164,15 @@ def map_maintained_school_status(
 
 
 def map_school_website(url: str):
-    if url is None:
+    if url is None or url == "":
         return None
     if url.startswith("https://") or url.startswith("http://"):
         return url
     return "https://" + url
 
 
-def map_is_pfi_school(pfi: str):
-    return (
-        "PFI School"
-        if pfi is not None and pfi.lower() == "part of pfi"
-        else "Non-PFI school"
-    )
+def map_is_pfi_school(pfi: float):
```

Comment:
nitpick: if this is possible, it should be `url: float | None` above.

Reply:
Will sort in a following merge